src/app/screens/TodoList/components/Todo/index.js

```diff
@@ -0,0 +1,18 @@
+import { View, Text, Button } from 'react-native';
+import React from 'react';
+
+import { DONE, DELETE } from '../../strings';
+
+import styles from './styles';
+
+export default function Todo({ onDelete, onToggleComplete, toDo }) {
+  return (
+    <View style={styles.toDoContainer}>
+      <Text style={toDo.completed && styles.lineThroughText}>{toDo.text}</Text>
+      <View style={styles.toDoOptions}>
+        <Button onPress={() => onToggleComplete(toDo.id)} title={DONE} />
```

Comment:
change this function 'Todo' by a class and move `onToggleComplete` and `onDelete` like a function of the class
```
handleToggleComplete = () => {
   this.props.onToggleComplete(this.props.toDo.id)
}
```

So you are avoiding create a new function reference every render call